src/generators.rs

```diff
@@ -153,34 +153,14 @@ impl BulletproofGens {
     /// * `party_capacity` is the maximum number of parties that can
     ///    produce an aggregated proof.
     pub fn new(gens_capacity: usize, party_capacity: usize) -> Self {
-        use byteorder::{ByteOrder, LittleEndian};
-
-        BulletproofGens {
+        let mut gens = BulletproofGens {
             gens_capacity,
```

Comment:
Oh, another thought: if you set initial `gens_capacity` to 0 instead of whatever user asked for, then you can avoid adding a helper `compute_generators` and instead directly call `increase_capacity()` with the intended capacity.

Reply:
sorry for my misleading suggestion earlier with skip/take arguments.